mmv1/third_party/terraform/services/netapp/resource_netapp_volume_test.go

```diff
@@ -659,3 +668,31 @@ func testAccNetappVolume_volumeBasicExample_cleanupScheduledBackup(t *testing.T,
 		return nil
 	}
 }
+
+func testAccNetappVolume_volumeBasicExample_autoTier(context map[string]interface{}) string {
+	return acctest.Nprintf(`
+resource "google_netapp_storage_pool" "default" {
+    name = "tf-test-pool%{random_suffix}"
+    location = "us-west2"
+    service_level = "PREMIUM"
+    capacity_gib = "2048"
+    network = data.google_compute_network.default.id
+    allow_auto_tiering = true
```

Comment:
Same here, we probably want a dedicated test. Since the new fields in `google_netapp_volume` we'll want to test a couple of different values from them in different `Config` steps, though.

Reply:
Done

Reply:
Ah, this was missing an important word, whoops. Since the new fields are updatable we'll need to add an update test (i.e. a test with multiple `Config` steps that exercise different values), as is covered at https://googlecloudplatform.github.io/magic-modules/develop/test/test/#add-an-update-test

I renamed the test for this resource based on it having a single `Config` step below; you probably want to undo that and add more steps.